streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBWindowStore.java

```diff
@@ -77,17 +76,17 @@ public void openDB(final ProcessorContext context) {
 
     private static class RocksDBWindowStoreIterator<V> implements WindowStoreIterator<V> {
         private final StateSerdes<?, V> serdes;
-        private final Iterator<Segment> segments;
+        private final Iterator<KeyValueStore<Bytes, byte[]>> segments;
```

Comment:
We can now that we moved the cache out of the WindowStore. Sorry, thought i'd changed that back